robotoff/insights/question.py

```diff
@@ -240,10 +242,7 @@ class LabelQuestionFormatter(QuestionFormatter):
 
     def format_question(self, insight: ProductInsight, lang: str) -> Question:
         value_tag: str = insight.value_tag
-        image_url = None
-
-        if value_tag in LABEL_IMAGES:
-            image_url = LABEL_IMAGES[value_tag]
+        ref_image_url = LABEL_IMAGES.get(value_tag)
```

Comment:
In the loaded dictionnary LABEL_IMAGES, is there all the labels in the taxonomy ? Because if it's not the case, how do we handle the case where value_tag is not among the dictionnary ?

Reply:
no there are only a few labels that have a logo. If the label ID is not in `LABEL_IMAGES`, None is passed as `ref_image_url`.